src/studio-header/header.scss

```diff
@@ -1,12 +1,33 @@
 // This SCSS was copied from edx/frontend-app-learning/src/index.scss.
 .course-title-lockup {
-    padding: 1rem 0;
-    padding-right: 1rem;
-    border-right: 1px solid #e5e5e5;
-    min-width: 0;
+  line-height: 1;
+  display: block;
+  padding: 0.5rem;
+  padding-right: 1rem;
+  border-right: 1px solid #e5e5e5;
+  min-width: 0;
     span {
+      color: #333;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
     }
 }
+
+.course-org {
```

Comment:
I think we can simplify this. Does the way the lockup in the learning MFE is designed work here?

[JSX](https://github.com/edx/frontend-app-learning/blob/2f01e8a64639e822c35333e7d7e2e6dd9ff4dcef/src/course-header/Header.jsx#L45)
[SCSS](https://github.com/edx/frontend-app-learning/blob/086b5d898611b424cec14bf85c63bebb0fa5c3aa/src/index.scss#L58)

Reply:
I didn't know those existed! Thanks for linking to them. I think standardizing styling for the lockup makes sense, so I can update the css to better reflect what already exists on the learning MFE.